platforms/nuttx/nuttx-configs/px4fmu-v2/nsh/defconfig

```diff
@@ -571,7 +571,8 @@ CONFIG_STM32_USART_SINGLEWIRE=y
 # SPI Configuration
 #
 # CONFIG_STM32_SPI_INTERRUPTS is not set
-# CONFIG_STM32_SPI_DMA is not set
+CONFIG_STM32_SPI_DMA=y
+CONFIG_STM32_SPI4_DMA=y
```

Comment:
The newer PX4 boards use px4fmu-v3, the only difference I am aware of is it has more onboard memory, so hopefully we can make the same changes there.

Reply:
In the internal build structure of px4, the px4fmu-v3 architecture uses the px4fmu-v2 definitions - so these changes automatically apply to anything on the px3fmu-v3 build chain too.